charts/cloudzero-agent/templates/cm.yaml

```diff
@@ -33,6 +33,9 @@ data:
         enable_compression: true
         follow_redirects: true
         enable_http2: true
```

Comment:
You'll want the entire section to look like this, the existing rules will not work as you'd like - they are designed for Service discovery. These https://github.com/Cloudzero/cloudzero-agent-validator/blob/33d5ebb337a2d67019d5fed37611e544a1ee1bf8/pkg/cmd/config/internal/scrape_config.tmpl#L67-L118 will be what you want

Reply:
Awesome thanks, makes sense.